Sourcery/Templates/Diffable.stencil

```diff
@@ -2,12 +2,13 @@
 extension {{ type.name }}{% if not type.supertype.implements.AutoDiffable %}: Diffable{% endif %} {
     {% if type.based.Type %}override {% endif %}func diffAgainst(_ object: Any?) -> DiffableResult {
         let results = DiffableResult()
-        {% for variable in type.storedVariables %}{% if not variable.annotations.skipEquality %}{% if forloop.first %}guard let rhs = object as? {{ type.name }} else {
+        guard object is {{ type.name }} else {
```

Comment:
Just noticed that you already updated Stencil to the commit with filters support for if tag, so my previous comment is not relevant any more 😀